src/plugin/timer/timerwrappers.cpp

```diff
@@ -124,13 +125,14 @@ clock_getcpuclockid(pid_t pid, clockid_t *clock_id)
 extern "C" int
 pthread_getcpuclockid(pthread_t thread, clockid_t *clock_id)
 {
-  DMTCP_PLUGIN_DISABLE_CKPT();
+  // We need to acquire exclusive lock here because the corresponding Pid plugin
```

Comment:
Picky, but:
'to acquire _an_ exclusive lock'
'plugin wrapper requires _an_ exclusive lock'